chain/network/tests/announce_account.rs

```diff
@@ -415,3 +415,86 @@ fn test_infinite_loop() {
     })
     .unwrap();
 }
+
+#[test]
```

Comment:
Looks to me that this test can be refactored/integrated with `test_infinite_loop`.

Reply:
Second test was enough for both targets. Left only one.